types/vee-validate.d.ts

```diff
@@ -150,6 +150,7 @@ export class Validator {
     validate(name: string, value?: any, scope?: string, silent?: boolean): Promise<any>;
     validateAll(values?: Object, scope?: string, silent?: boolean): Promise<any>;
     validateScopes(silent?: boolean): Promise<any>;
+    verify(value: any, rules: string|Object): Promise<any>;

```

Comment:
You could enhance it further, since the `verify` method will always return a promise that always resolves to:

```
 { valid: boolean, errors: string[] }
```